packages/apps/reputation-oracle/server/src/modules/user/user.entity.ts

```diff
@@ -48,4 +48,7 @@ export class UserEntity extends BaseEntity implements IUser {
     (userQualification) => userQualification.user,
   )
   public userQualifications: UserQualificationEntity[];
+
+  @Column({ type: 'varchar', nullable: true })
+  public ndaSigned?: string;
```

Comment:
I'd rename `ndaSigned` to something like `ndaSignedUrl` or something similar because `ndaSigned` name makes me think that we store a boolean type.